web/styles/zulip.css

```diff
@@ -1298,6 +1298,10 @@ div.toggle_resolve_topic_spinner .loading_indicator_spinner {
 
 .empty-feed-notice-description {
     font-size: 1.1em;
+
+    span {
```

Comment:
We don't use CSS selectors like this that don't have a class on them. If you need to edit the HTML to add a class, taht's fine.

(It causes performance issues in addition to being fragile and hard to read).